packages/terra-hookshot/src/terra-dev-site/doc/example/BoundedHookshotExample.jsx

```diff
@@ -109,35 +106,32 @@ class HookshotStandard extends React.Component {
 
     return (
       <div>
-        <form>
-          <label htmlFor={this.getId('hookshotAttachmentBehavior')}>Attachment Behavior</label>
-          <select id={this.getId('hookshotAttachmentBehavior')} name="hookshotAttachmentBehavior" value={this.state.hookshotAttachmentBehavior} onChange={this.handleSelectChange}>
-            {generateOptions(Hookshot.attachmentBehaviors)}
-          </select>
-          <br />
-          <br />
-          <NumberField
-            label="Attachment Margin in Px"
-            id={this.getId('hookshotAttachmentMargin')}
-            name="hookshotAttachmentMargin"
-            defaultValue={0}
-            value={this.state.hookshotAttachmentMargin}
-            style={{ width: '200px' }}
-            onChange={this.handleInputChange}
-          />
-          <label htmlFor={this.getId('hookshotContentAttachment')}>Content Attachment</label>
-          <select id={this.getId('hookshotContentAttachment')} name="hookshotContentAttachment" value={this.state.hookshotContentAttachment} onChange={this.handleSelectChange}>
-            {generateOptions(ATTACHMENT_POSITIONS)}
-          </select>
-          <br />
-          <br />
-          <label htmlFor={this.getId('hookshotTargetAttachment')}>Target Attachment</label>
-          <select id={this.getId('hookshotTargetAttachment')} name="hookshotTargetAttachment" value={this.state.hookshotTargetAttachment} onChange={this.handleSelectChange}>
-            {generateOptions(ATTACHMENT_POSITIONS)}
-          </select>
-          <br />
-          <br />
-        </form>
+        <label htmlFor={getId('hookshotAttachmentBehavior')}>Attachment Behavior</label>
+        <select id={getId('hookshotAttachmentBehavior')} name="hookshotAttachmentBehavior" value={this.state.hookshotAttachmentBehavior} onChange={this.handleSelectChange}>
+          {generateOptions(Hookshot.attachmentBehaviors)}
+        </select>
+        <br />
+        <br />
+        <InputField
+          label="Attachment Margin in Px"
+          inputId={getId('hookshotAttachmentMargin')}
+          inputAttrs={{ name: 'hookshotAttachmentMargin' }}
+          defaultValue={this.state.hookshotAttachmentMargin}
+          style={{ width: '200px' }}
+          onChange={this.handleInputChange}
+        />
+        <label htmlFor={getId('hookshotContentAttachment')}>Content Attachment</label>
```

Comment:
We do have select fields. I recommend using them here. https://engineering.cerner.com/terra-core/#/components/terra-form-select/form-select/select-field